examples/hybrid/comparison_plots.jl

```diff
@@ -0,0 +1,209 @@
+using Plots
+using Plots.PlotMeasures
+using JLD2
+
+include("cli_options.jl")
+if !(@isdefined parsed_args)
+    (s, parsed_args) = parse_commandline()
+end
+parse_arg(pa, key, default) = isnothing(pa[key]) ? default : pa[key]
+
+job_id =
+    isnothing(parsed_args["job_id"]) ? job_id_from_parsed_args(s, parsed_args) :
+    parsed_args["job_id"]
+figtypes = (".png", ".pdf") # format for saving scaling plots
+figtype = ".png"
+output_dir = "./"
+
+secs_per_hour = 60 * 60
+secs_per_day = 60 * 60 * 24
+days_per_year = 8760 / 24
+
+# tempest scaling data
+if occursin("low", job_id)
+    resolution = "low-resolution"
+    nprocs_tempest = [1, 2, 3, 6, 24]
+    walltime_tempest = [74.41848, 34.43472, 23.22432, 11.83896, 3.60936]
+    t_int_days_tempest = 10 # integration time for Tempest
+    t_int_days_climaatmos = 10 # integration time for ClimaAtmos
+else
+    resolution = "high-resolution"
+    nprocs_tempest = [1, 2, 3, 6, 24, 54, 96, 216]
+    walltime_tempest = [
+        4673.969568,
+        2338.740864,
+        1505.628864,
+        755.51184,
+        245.157408,
+        130.145184,
+        76.487328,
+        32.298912,
+    ]
+    t_int_days_tempest = 1 # integration time for Tempest
+    t_int_days_climaatmos = 1 # integration time for ClimaAtmos
+end
+t_int_days = t_int_days_climaatmos
+t_int = string(t_int_days_climaatmos) * " days"
+# normalalize Tempest walltime for comparison with ClimaAtmos (if necessary)
+walltime_tempest .*= (t_int_days / t_int_days_tempest)
+
+# read ClimaAtmos scaling data
+I, FT = Int, Float64
+nprocs_clima_atmos = I[]
+walltime_clima_atmos = FT[]
+for foldername in readdir(output_dir)
+    if occursin(job_id, foldername)
+        nprocs_string = split(foldername, "_")[end]
+        dict = load(
+            joinpath(
+                output_dir,
+                foldername,
+                "scaling_data_$(nprocs_string)_processes.jld2",
+            ),
+        )
+        push!(nprocs_clima_atmos, I(dict["nprocs"]))
+        push!(walltime_clima_atmos, FT(dict["walltime"]))
+    end
+end
```

Comment:
```suggestion
folders = readdir(output_dir)
filter!(f -> occursin(job_id, f), folders)
# Add debug for `folders`
files = map(folders) do folder
    joinpath(
        output_dir,
        folder,
        "scaling_data_$(last(split(folder, "_")))_processes.jld2",
    )
end
@assert all(isfile.(files))
dicts = map(f -> JLD2.load(f), files)
nprocs_clima_atmos = map(dict->Int(dict["nprocs"]), dicts)
walltime_clima_atmos = map(dict->Float64(dict["walltime"]), dicts)
```
Not that this is super important (this is fine as it is), but stylistically, I'd usually write this block a bit differently so that you can debug line-by-line a bit more easily for all of the jobs/files